docs/manual-CN/functions-and-operators/type-conversion.md

```diff
@@ -1,3 +1,17 @@
-<!---
-Not supported yet
---->
\ No newline at end of file
+# 类型转换
+
+Nebula Graph 中，类型转换分为隐式转换和显式转换。
+
+## 隐式转换
+
+在表达式中，兼容类型间可自动完成类型转换：
+
+1. 所有类型均可隐式转换至 `bool` 类型：
+  a. 当且仅当字符串长度为零时，可被隐式转换为 `false` ，否则为 `true`
+  b. 当且仅当整型数值为零时，可被隐式转换为 `false` ，否则为 `true`
+  c. 当且仅当浮点类型数值为 `0` 时，可被隐式转换为 `false` ，否则为 `true`
+2. `int` 类型可隐式转换为 `double` 类型
```

Comment:
use markdown 1.

Reply:
> 这个文件逻辑不是很顺，我感觉要重新描述：
> 可以说nGQL的类型转换规则与C语言相通，

Sorry, but I dont known what you are talking about.